src/documents/components/ReviewDetail/ReviewDetail.js

```diff
@@ -338,6 +340,88 @@ const ReviewDetail = ({
                   </Label>
                 </Segment>
               </Grid.Column>
+              <Grid.Column mobile={16} tablet={16} computer={7}>
+                <Segment basic>
+                  <Header as="h4" color="grey">
+                    Validation Summary
+                  </Header>
+                  {reviewNode.validationResultset ? (
+                    <Label.Group>
+                      <Label
+                        as="a"
+                        onClick={() => {
+                          history.push(
+                            `/study/${match.params.kfId}/reviews/${reviewNode.kfId}/validation`,
+                          );
+                        }}
+                      >
+                        <Icon name="check circle" color="green" />
+                        {reviewNode.validationResultset.passed} Passed
+                      </Label>
+                      <Label
+                        as="a"
+                        onClick={() => {
+                          history.push(
+                            `/study/${match.params.kfId}/reviews/${reviewNode.kfId}/validation`,
+                          );
+                        }}
+                      >
+                        <Icon name="times circle" color="red" />
+                        {reviewNode.validationResultset.failed} Failed
+                      </Label>
+                      <Label
+                        as="a"
+                        onClick={() => {
+                          history.push(
+                            `/study/${match.params.kfId}/reviews/${reviewNode.kfId}/validation`,
+                          );
+                        }}
+                      >
+                        <Icon name="warning circle" color="yellow" />
+                        {reviewNode.validationResultset.didNotRun} Did Not Run
+                      </Label>
+                    </Label.Group>
+                  ) : (
+                    <>
+                      {['in_review', 'awaiting_updates'].includes(
+                        reviewNode.state,
+                      ) && canStartValidation ? (
+                        <>
+                          {validationRunState === 'running' ? (
+                            <Label
+                              as="a"
+                              color="blue"
+                              onClick={() => {
+                                history.push(
+                                  `/study/${match.params.kfId}/reviews/${reviewNode.kfId}/validation`,
```

Comment:
I don't know much about Semantic UI - does it not allow you to pass `href` here?